src/mame/nmk/nmk16.cpp

```diff
@@ -9203,35 +9209,35 @@ GAME( 1991, tdragonb2,  tdragon,  tdragonb2,    tdragon,      nmk16_state, empty
+GAME( 1997, redhawksa,  stagger1, redhawki,     stagger1,     afega_state, init_redhawksa,       ROT0,               "Afega (Hae Dong Corp license)",     "Red Hawk (horizontal, Spain, set 2)", MACHINE_NO_COCKTAIL )
+GAME( 1997, redhawkg,   stagger1, redhawki,     stagger1,     afega_state, init_redhawkg,        ROT0,               "Afega",                             "Red Hawk (horizontal, Greece)", MACHINE_NO_COCKTAIL )
+GAME( 1997, redhawke,   stagger1, stagger1,     stagger1,     afega_state, empty_init,           ROT270,             "Afega (Excellent Co. license)",     "Red Hawk (Excellent Co., Ltd)", MACHINE_NO_COCKTAIL ) // earlier revision? different afega logo and score and credit number fonts compared to other sets
+GAME( 1997, redhawkk,   stagger1, stagger1,     stagger1,     afega_state, empty_init,           ROT270,             "Afega",                             "Red Hawk (Korea)", MACHINE_NO_COCKTAIL )
+GAME( 1997, redhawkb,   stagger1, redhawkb,     redhawkb,     afega_state, empty_init,           ROT0,               "bootleg (Vince)",                   "Red Hawk (horizontal, bootleg)", MACHINE_NO_COCKTAIL )
+
+GAME( 1998, grdnstrm,   0,        grdnstrm,     grdnstrm,     afega_state, empty_init,           ORIENTATION_FLIP_Y, "Afega (Apples Industries license)", "Guardian Storm (horizontal, not encrypted)", MACHINE_NO_COCKTAIL )
+GAME( 1998, grdnstrmv,  grdnstrm, grdnstrmk,    grdnstrk,     afega_state, init_grdnstrm,        ROT270,             "Afega (Apples Industries license)", "Guardian Storm (vertical)", MACHINE_NO_COCKTAIL )
+GAME( 1998, grdnstrmj,  grdnstrm, grdnstrmk,    grdnstrk,     afega_state, init_grdnstrmg,       ROT270,             "Afega",                             "Sen Jing - Guardian Storm (Japan)", MACHINE_NO_COCKTAIL )
+GAME( 1998, grdnstrmk,  grdnstrm, grdnstrmk,    grdnstrk,     afega_state, init_grdnstrm,        ROT270,             "Afega",                             "Jeon Sin - Guardian Storm (Korea)", MACHINE_NO_COCKTAIL )
+GAME( 1998, redfoxwp2,  grdnstrm, grdnstrmk,    grdnstrk,     afega_state, init_grdnstrm,        ROT270,             "Afega",                             "Hong Hu Zhanji II (China, set 1)", MACHINE_NO_COCKTAIL )
+GAME( 1998, redfoxwp2a, grdnstrm, grdnstrmk,    grdnstrk,     afega_state, init_redfoxwp2a,      ROT270,             "Afega",                             "Hong Hu Zhanji II (China, set 2)", MACHINE_NO_COCKTAIL )
+GAME( 1998, grdnstrmg,  grdnstrm, grdnstrmk,    grdnstrk,     afega_state, init_grdnstrmg,       ROT270,             "Afega",                             "Guardian Storm (Germany)", MACHINE_NO_COCKTAIL )
+GAME( 1998, grdnstrmau, grdnstrm, grdnstrm,     grdnstrm,     afega_state, init_grdnstrmau,      ORIENTATION_FLIP_Y, "Afega",                             "Guardian Storm (horizontal, Australia)", MACHINE_NO_COCKTAIL )
```

Comment:
Are these games supposed to support cocktail cabinets at all?  A quick look at the DIP switches for Stagger 1 and Red Hawk doesn’t show a cabinet type setting.  The `MACHINE_NO_COCKTAIL` flag is only supposed to be used if the game supports cocktail cabinets but MAME doesn’t emulate the feature correctly.  It isn’t for games that just lack support for cocktail cabinets.

Reply:
Yes, you're right, they doesn't have specific option for cabinet setting, but they have flip-screen option that doesn't work fine, that's why added that flag. 
Is there any other flag that is more appropriate to mark this situation?

Reply:
We don’t have a specific flag for that.  You could open a bug at MAME Testers to track it if there isn’t one, or you could put a comment in the source.

As an aside, that specific flag seems misguided at best.  At a guess, it was probably added in response to a large number of bug reports about cocktail mode not working when MAME Testers had a blanket policy of not accepting reports for flagged issues.  In retrospect, it seems overly specific.  With MAME’s expanded scope, it isn’t relevant to a lot of systems.  Given how few people use MAME in cocktail cabinets (ignoring MAME-based bootleg systems in restaurants and bars), I don’t think it’s the most useful thing to be able to display or filter on in front-ends either.  It isn’t even that useful for compiling a list of games suitable for a cocktail cabinet as it gives no indication when a system doesn’t support cocktail cabinets.

Reply:
Careful what you wish for – that could quickly get applied to just about every system in MAME.

Reply:
Removed `MACHINE_NO_COCKTAIL` flag for `stagger1`, `grndstrm` and clones